src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -764,7 +853,7 @@ The infrastructure parameters are described hereafter:
  ** `sudo`: similar to `none` regarding the connection to the head node, but a `sudo` command will be initiated to impersonate as the current <<_glossary_proactive_scheduler_user,ProActive Scheduler user>>, before doing a job submission.
  * *alternateRMUrl* : the url used by the ProActive Nodes to contact <<_glossary_resource_manager,ProActive Resource Manager>>. This url is displayed on ProActive server startup. Example: `pnp://myserver:64738`.
  * *sshPort* : port used for ssh connections.
```

Comment:
`ssh` => `SSH`

Reply:
Agree, there is also another lowercase "url". Let me piggyback all these corrections.